MekHQ/src/mekhq/campaign/parts/ProtomekSensor.java

```diff
@@ -72,7 +70,7 @@ public Money getStickerPrice() {
     @Override
     public boolean isSamePartType (Part part) {
         return part instanceof ProtomekSensor
-                && getUnitTonnage() == ((ProtomekSensor)part).getUnitTonnage();
+                && getUnitTonnage() == part.getUnitTonnage();
```

Comment:
Double check that ProtomekSensor doesn't have an implementation of getUnitTonnage()

Reply:
Same as above